docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md

```diff
@@ -380,6 +240,8 @@ To create a YugabyteDB Anywhere instance, perform the following:
   - ClusterRole or ClusterRoleBinding with the cluster-admin role are recommended if your intention is to create clusters across multiple namespaces.
```

Comment:
```suggestion
Creating ClusterRole or ClusterRoleBinding with the cluster-admin role are recommended if your intention is to create clusters across multiple namespaces.

```

Reply:
Actually, this sentence shouldn't be here. I will remove it.